.ci/scripts/compare_dirs.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -eux
+
+# Check if dir1's files are also found in dir2 with the same
+# contents. Exempt files named BUCK, CMakeLists.txt, TARGETS, or
+# targets.bzl.
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 dir1 dir2" >&2
+    exit 1
+fi
+dir1="$1"
+dir2="$2"
+
+if [ ! -d "$dir1" ] || [ ! -d "$dir2" ]; then
+    echo "Error: Both directories must exist" >&2
+    exit 1
+fi
+
+while IFS= read -r -d '' file; do
```

Comment:
Thoughts on continuing after errors (instead of exiting on first mismatch)

Reply:
done